cmd/seed-controller-manager/options.go

```diff
@@ -155,6 +159,9 @@ func newControllerRunOptions() (controllerRunOptions, error) {
 	flag.IntVar(&c.schedulerDefaultReplicas, "scheduler-default-replicas", 1, "The default number of replicas for usercluster schedulers")
 	flag.IntVar(&c.concurrentClusterUpdate, "max-parallel-reconcile", 10, "The default number of resources updates per cluster")
 	flag.IntVar(&c.addonEnforceInterval, "addon-enforce-interval", 5, "Check and ensure default usercluster addons are deployed every interval in minutes. Set to 0 to disable.")
+	flag.IntVar(&c.controlPlaneLeaderElectLeaseDurationSeconds, "control-plane-leader-elect-lease-duration", 0, "The lease duration in seconds used by control plane components using leader election (i.e. controller-manager and schedule). The default value for the component is used when equal or less than 0.")
```

Comment:
Should be more appropriate to call it `user-cluster-leader-elect-lease-duration`?

Reply:
I don't have strong opinions. Maybe `user-cluster-leader-elect-lease-duration` would be better and more clear.

Reply:
yeah, I have the same feeling